metadata/v20/v20/schema.json

```diff
@@ -892,9 +900,23 @@
                     "string",
                     "null"
                   ],
-                  "example": "\u00a9 Reiner Lemoine Institut",
+                  "examples": [
+                    "\u00a9 Reiner Lemoine Institut"
```

Comment:
Here seems to be an issue with asci codes? But i think this was resolved. Note to me to check it later.